SPEC.md

```diff
@@ -549,6 +549,15 @@ The AC Name Type is used as the primary key for a number of fields in the schema
 The schema validator will ensure that the keys conform to these constraints.
 
 
+## AC Version Type
+
+The application container specification aims at being a forward compatible specification. Where as you are complaint to version 1.x, then your application will maintain compatibility in the 1.x major version series.
+
+The current version of the application container spec is recorded in [VERSION](https://github.com/appc/spec/blob/master/VERSION), and is otherwise denoted in the [release version](https://github.com/appc/spec/releases) or git version control tag. 
```

Comment:
We are (almost completely) consistent on "App Container" (rather than "application container")